.travis.yml

```diff
@@ -36,5 +36,5 @@ services:
   - postgresql
 
 before_script:
-  - mysql -e 'create database ancestry_test;' || true
-  - psql -c 'create database ancestry_test;' -U postgres || true
+  - mysql -e 'CREATE DATABASE ancestry_test CHARACTER SET utf8 COLLATE utf8_general_ci;' || true
```

Comment:
Hi @antstorm 

I think this line is significant. Could you please explain more about this?

We may want to add this to the general documentation to help people.
Also if this is not changed as the database collation, then I wonder what users can do to get this behavior.

I'm also thinking postgres on mac has a similar issue with this.

Reply:
or maybe this line should be collation on just the column if they don't do this to the whole database?

Reply:
I added collation to the column and not to the whole database.
Thanks for putting this on my radar.